loleaflet/src/control/Control.NotebookbarDraw.js

```diff
@@ -191,9 +191,9 @@ L.Control.NotebookbarDraw = L.Control.NotebookbarImpress.extend({
 						'children': [
 							{
 								'id': 'print',
-								'type': 'menubartoolitem',
+								'type': 'bigtoolitem',
 								'text': _UNO('.uno:Print', 'presentation'),
-								'command': ''
+								'command': '.uno:Print'
```

Comment:
hi, we shouldn't execute Print uno command. we need to execute JS code which will trigger browser's print dialog. (we don;t want to print on a printer connected to the collabora server, but using client's printer :) )

Reply:
nevermind, I see this command is overriden in NotebookbarBuilder :)

Reply:
yes it is that's how I get rid of ellipsis :) thanks anyhow for checking it, I'm adding now one more commit to this PR